README.md

```diff
@@ -54,9 +55,9 @@ returns business logic version
 ## Configuration
 The ECC supports three different way to exchange data:
 
-*  **REST endpoints** enabled if *IDSCP2=false* and *WS_OVER_HTTPS=false*
-*  **IDSCP2** enabled if *IDSCP2=true* and *WS_INTERNAL=false* (use https on the edge) or *IDSCP2=true* and *WS_INTERNAL=true* (use WS on the edge)
-*  **Web Socket over HTTPS** enabled if *WS_OVER_HTTPS=true* and *IDSCP2=false*
+*  **REST endpoints** enabled if *WS_EDGE=false* and *WS_ECC=false*
+*  **IDSCP2** enabled if *IDSCP2=true* and *WS_EDGE=true* (use websocket on the edge, false for REST on the edge) and  *WS_ECC=true* (use ws on between connectors, false for REST between connectors)
```

Comment:
It's not clear that we can use https on the edge then idscp and how to set it up

Reply:
Does this makes more sense?
IDSCP2 enabled if IDSCP2=true and WS_ECC = false 
For WS_EDGE=true (use websocket on the edge, false for REST on the edge)